src/locales/en/translation.json

```diff
@@ -225,7 +230,7 @@
         "paragraph3": "The code will expire after  15 minutes.",
         "requestNewCodeLink": "Request a new code",
         "requestNewCodeParagraph": " if:",
-        "requestNewCodeReason1": "the code isn’t working or you didn’t receive one",
+        "requestNewCodeReason1": "the code isn’t working or has expired, or you didn’t receive one",
```

Comment:
Too many "or"s, should this be "isn't working, has expired or you didn't receive one"?

Reply:
taken from figma so i assume thats the truth